### PR TITLE
Improvements to Player Input component docs

### PR DESCRIPTION
**Pull request by duckets**
### Description

Clarified information about accessing actions via script (ISX-1758) and updated some wording and screenshots to reflect project-wide actions as the default workflow.

https://jira.unity3d.com/browse/ISX-1758

### Changes made

Added notes to Player Input component docs, and Player Input workflow page, and updated screenshots.

### CODE REVIEW

File: Packages/com.unity.inputsystem/Documentation~/PlayerInput.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/PlayerInput.md]: I think you need a PlayerManager Component to do the paring of each player to a unique set of devices. So I think this is wrong when using the PlayerInput component on its own.

File: Packages/com.unity.inputsystem/Documentation~/PlayerInput.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/PlayerInput.md:160]: Great - thanks for adding

**Review by lyndon-unity — CHANGES_REQUESTED** [review]
Looks good but you need to run the formatting script to fix it before the tests will pass
I use powershell and run .\format.ps1

File: Packages/com.unity.inputsystem/Documentation~/PlayerInput.md
Comment (ekcoh) [reply Packages/com.unity.inputsystem/Documentation~/PlayerInput.md]: You do not need a PlayerManager to achieve the pairing. I believe just need to have multiple PlayerInput component instances. PlayerManager is needed to get the "press to join behavior". Have you observed something different @lyndon-unity ?

File: Packages/com.unity.inputsystem/Documentation~/PlayerInput.md
Comment (ekcoh) [reply Packages/com.unity.inputsystem/Documentation~/PlayerInput.md:160]: Agreed! Thanks for trying to avoid this pitfall via doc guidance

**Review by ekcoh — APPROVED** [review]
LGTM if points by @lyndon-unity are addressed. Hence I will approve since Lyndon already requested changes which blocks the merge until adressed.

File: Packages/com.unity.inputsystem/Documentation~/PlayerInput.md
Comment (lyndon-unity) [reply Packages/com.unity.inputsystem/Documentation~/PlayerInput.md]: Joao tried using player component on its own (multiple instances in the scene) and each responded to all inputs (e.g. gamepad drove all instances). There was no automatic filtering based on which player started the interaction.
I think to get that filtering you have to call some specific functions to choose which devices to filter to. I.e. its not automatic unless using the PlayerInputManager. This definitely confused Joao so I think the above info is needed (or some enhanced variant)

File: Packages/com.unity.inputsystem/Documentation~/PlayerInput.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/PlayerInput.md:31]: I think we should say something about how it pairs them.
I think its this (but I'd like Hakan's input as I might not be 100% correct)

The system looks up the 'control scheme' associated with the control that is pressed.
If one is found it makes that the only active control scheme applicable for that player.

It also looks at the specific device that the control was pressed on (e.g. The first gamepad connected) and binds that to be the only device associated with the player.

... then we can explain that the WASD could be in control scheme named 'left keyboard' and cursors in a control scheme named 'right keyboard' and two players would be able to play on that same keyboard device.

**Review by lyndon-unity — APPROVED** [review]
Looks good - I've suggest a further enhancement.

**Comment by lyndon-unity** [comment]
Need to run the ./format.ps1 command in powershell

Packages/com.unity.inputsystem/Documentation~/PlayerInput.md
[14:23:05.303 Information] -> 1 file(s) need formatting changes.
...599 total files were examined

**Comment by lyndon-unity** [comment]
Also looks like this page needs a fix

Message: Expected: <empty> But was: < "ActionsAssets.md in C:\build\output\Unity-Technologies\InputSystem\Temp\docstest\com.unity.inputsystem@1.8\manual\PlayerInput.html (File Not Found)" >

**Comment by lyndon-unity** [comment]
> Also looks like this page needs a fix
> 
> Message: Expected: But was: < "ActionsAssets.md in C:\build\output\Unity-Technologies\InputSystem\Temp\docstest\com.unity.inputsystem@1.8\manual\PlayerInput.html (File Not Found)" >

Looks like ActionsAssets should be ActionAssets - drop the s

File: Packages/com.unity.inputsystem/Documentation~/PlayerInput.md
Comment (lyndon-unity) [reply Packages/com.unity.inputsystem/Documentation~/PlayerInput.md]: Ignore this comment as further testing confirmed device binding does work without player input manager.

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/InputSystem?pullRequest=1827) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/InputSystem?pullRequest=1827) before we can accept your contribution.<br/><hr/>**Lyndon Homewood** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/InputSystem?pullRequest=1827) it.</sub>

**Pull request merged**
